lte/gateway/c/session_manager/GrpcMagmaUtils.cpp

```diff
@@ -79,3 +79,9 @@ std::string indentText(std::string basicString, int indent) {
   }
   return result;
 }
+
+void prepend_imsi_with_prefix(std::string& imsi) {
+  if (imsi.find("IMSI") == std::string::npos) {
```

Comment:
Let's define a macro for IMSI_PREFIX here.

Reply:
Defined macro `IMSI_PREFIX` in `Utilities.h`